drivers/serial/uart_rpmsg.c

```diff
@@ -412,8 +412,14 @@ int uart_rpmsg_init(FAR const char *cpuname, FAR const char *devname,
       return ret;
     }
 
+  /* Ignore the `isconsole` to passthrough all UART data,
+   * since the most common usage of rpmsg uart is to communicate with cu
+   * in other processor core, should not do CRLF convertion and ECHO.
+   * Note: you can also change this behavior by termios.
+   */
+
   dev->ops       = &g_uart_rpmsg_ops;
-  dev->isconsole = isconsole;
```

Comment:
let remove isconsole argument from uart_rpmsg_init, or modify the caller to pass false.

Reply:
Ok, I forget this.